db_migrate/versions/e89887e7d3f0_add_bug_path_length.py

```diff
@@ -0,0 +1,33 @@
+"""Add bug path length
+
+Revision ID: e89887e7d3f0
+Revises: 3793e361a752
+Create Date: 2018-08-28 11:11:07.533906
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e89887e7d3f0'
+down_revision = '3793e361a752'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+
+    op.add_column('reports',
+                  sa.Column('path_length', sa.Integer(), nullable=True, default=0))
```

Comment:
I think we always come back to this at such changes, but SQLAlchemy still don't support views, right?

Nontheless, I am worried about allowing `nullable=True` here. The rest looks good.